apps/frontend/src/pages/project/project.tsx

```diff
@@ -37,6 +37,12 @@ const Project = (): JSX.Element => {
 
 	return (
 		<PageLayout isLoading={isLoading}>
+			<Breadcrumbs
+				items={[
+					{ href: "/", label: "Projects" },
+					{ label: project?.name || "" },
```

Comment:
We know that we will always have project here. Let's not add additional checks just for typescript
```suggestion
					{ label: (project.name as ProjectType) },
```

Reply:
maybe just ```
{project && (
					<Breadcrumbs
						items={[{ href: "/", label: "Projects" }, { label: project.name }]}
					/>
				)}
```?